.github/mlc_config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^\.\./\.\./actions/workflows"
```

Comment:
This ignores the badge URLs, which seem to not check out (probably because of the relative paths?).